README.md

```diff
@@ -1,13 +1,55 @@
+- **Coaches:** Coaches help you to teach the attendees. One coach for two attendees is a good ratio.
+- **Attendees:** The attendees are the most important part of the workshop. Invite them via mailing lists, social media, posters, flyers etc. Attendees should be women or people from underrepresented groups in tech. Read more about [men’s attendance here](https://github.com/ClojureBridge/organizing/blob/master/Guidance-to-Mens-Attendance.md).
+
+We recommend you to join our [workshop mailing list](https://groups.google.com/forum/#!forum/clojurebridge-workshops). When you decided for a date and have found a venue we are happy to receive your information via an issue in our [workshops repository](https://github.com/ClojureBridge/Workshops) on GitHub. Then we will publish your workshop on our website and assign you a mentor who can help you with answers to questions and by providing resources and information.
+
+You can find lots of information in our [organizing repository](https://github.com/ClojureBridge/organizing) on GitHub and read about the [Workshop Planning Timeline](Workshop-Planning-Timeline.md). [Resources on Diversity](Resources-on-Diversity.md) provides a number of great resources on how to create a diverse, inclusive event. Feel free to read and contribute to these documents.
+
+ClojureBridge follows the BridgeFoundry [Code of Conduct](http://bridgefoundry.org/code-of-conduct/).
+
+Last, but not least, you might want to find some sponsors. They can help you provide awesome food, drinks or fancy swag for your learners so that this workshop will stay a memorable event.
+
+###...coach!
+
+As a coach you and your co-coach will teach a small group of attendees Clojure and fundamental programming principles at the workshop.This way you can help to start new carriers in tech. Have a look at our curriculum. Coaching is an awesome experience. It helps you understand Clojure better and to practice your explaining skills which is also useful for work.
+
+###...be a teaching assistant!
+
+As a teaching assistant or a “floating coach” you also help attendees learn Clojure. You are not assigned to a fixed group but you can float around and help when questions arise. Being a teaching assistant might be your thing if you just started learning Clojure yourself. 
+
+###...sponsor or donate!
+
+What would ClojureBridge be without our sponsors? Many great companies and private people have sponsored ClojureBridge workshops. With their help it was possible to provide food, drinks, T-Shirts, notebooks and stickers for the attendees at different workshops. 
+You can help by sponsoring the workshop in your area or abroad but also a donation to ClojureBridge is very welcome to help us spread the word, run our servers etc.
+
+###...help improve the curriculum!
+
+Our curriculum can be found in our curriculum repository. You are very welcome to join our curriculum mailing list and share your ideas.

```

Comment:
I actually had that in my draft and then forgot. Will put it in again. Thanks for the reminder. I will also add a few more links.